lib/mix/tasks/popular_packages.ex

```diff
@@ -0,0 +1,64 @@
+defmodule Mix.Tasks.PopularPackages do
+  use Mix.Task
+
+  @shortdoc "Rebuild popular_packages.txt"
+
+  @moduledoc """
+  Fetches a list of most downloaded packages from hex.pm and stores into the
+  top-level popular_packages.txt file.
+
+  ## Usage
+
+      mix popular_packages
+  """
+
+  @base_url "https://hex.pm/api/packages"
+  @file_path File.cwd!() <> "/popular_packages.txt"
+  @page_count 11
```

Comment:
I chose 1,100 packages (`@page_count` of 11) quite arbitrarily. Seems a reasonable default.

I'll happily merge this PR with a hard-coded value of 11 here, but it would be even better if your Mix task let the caller specify a different value for their personal use. Just an idea. Feel free to ignore.